packages/relay/tests/lib/eth.spec.ts

```diff
@@ -911,6 +911,7 @@ describe('Eth calls using MirrorNode', async function () {
     mock.onGet(`network/fees`).reply(200, defaultNetworkFees);
     const feeHistory = await ethImpl.feeHistory(1, 'latest', [25, 75]);
     expect(feeHistory).to.exist;
+    if (feeHistory == null) return;
```

Comment:
Q: Why is this and the check below on line 927 necessary? 
It shouldn't be hitting the backup flow where the sdkClient is used.

Reply:
Intellij's typescript compiler requires it.  The compiler cannot prove that the 913 line will keep nulls from passing, so at 914 it may be null.  This catch guarantees it.